apps/blockchain/lib/blockchain/contract/create_contract.ex

```diff
@@ -7,28 +7,32 @@ defmodule Blockchain.Contract.CreateContract do
   alias Blockchain.Interface.{BlockInterface, AccountInterface}
   alias Block.Header
   alias Blockchain.Contract.Address
-  alias Blockchain.{Account, Contract}
+  alias Blockchain.Account
   alias EVM.{SubState, Gas}
 
-  # σ
   defstruct state: %{},
-            # s
             sender: <<>>,
-            # o
             originator: <<>>,
-            # g
             available_gas: 0,
-            # p
             gas_price: 0,
-            # v
             endowment: 0,
-            # i
             init_code: <<>>,
-            # e
             stack_depth: 0,
             block_header: nil,
             config: EVM.Configuration.Frontier.new()
 
+  @typedoc """
+  Yello Paper Terms:
```

Comment:
Yello -> Yellow

Reply:
Good catch. Updated in bd04e6984ecbc935d77f745c4a9561bfc88d9c48